pkgs/applications/networking/irc/srain/default.nix

```diff
@@ -48,6 +49,7 @@ stdenv.mkDerivation rec {
     libsoup
     libsecret
     openssl
+    gnome.adwaita-icon-theme
```

Comment:
Icon themes are no longer picked up by `wrapGAppsHook` by default, so this won't work any more (#163623).

```suggestion
```
*(Also remove `gnome` from args)*

If the icon is hardcoded by the app (sample case [here](https://github.com/alainm23/planner/blob/9d1c710e2b528c12fa4e28b84035a838d3153d05/src/Application.vala#L137)), you need to follow the [manual](https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-packaging) to prefix `XDG_DATA_DIRS` with `XDG_ICON_DIRS`.